README.md

```diff
@@ -26,6 +28,11 @@
 
 ## Installation
 
+### Swift Package Manager
+
+**Swift Package Manager (requires Xcode 11)**
+Add package into Project settings -> Swift Packages -> Repo URL
+
```

Comment:
We already support min Xcode 11, therefore no need to mention `(requires Xcode 11)`.

What about,

1. File > Swift Packages > Add Package Dependency
2. Add `https://github.com/Retro-Cartoons/Tom.git`

_OR_

Update `dependencies` in `Package.swift`
```swift
dependencies: [
    .package(url: "https://github.com/Retro-Cartoons/Tom.git", .upToNextMajor(from: "1.0.0"))
]
```

Reply:
You're absolutely right. It's more understandable that way.